requirements.txt

```diff
@@ -11,6 +11,7 @@ configobj>=5.0.6
 networkx>=2.1
 pyyaml>=3.12
 gitpython>=2.1.8
+python-hglib==2.6.1
```

Comment:
You also need to add it to `install_requires` in `setup.py`. Btw, can we make a requirement less strict with `>=`?